src/styles/base.js

```diff
@@ -1114,6 +1115,7 @@ export const theme = {
   selectFontWeight,
   selectColorLT,
   selectFontWeightBold,
+  panelTabColor,
```

Comment:
you need to also add `panelTabColor` to  `themeLT` and `themeBS` and have it equals to the value of `subtextColor` in that theme

Reply:
Nice catch!. as both themes extends the theme variable I've added to themeLT and themeBS just the cases where that value was overwritten